### DISCUSSION TITLE
How to integrate with Azure Devops?

### DISCUSSION THREAD

**Discussion by hager-yousri** [reportportal/reportportal#1981] [Q&A] [answered]
I use Azure Devops to run pipelines and post results on Report portal 
I see that I can setup plugin to be able to create bugs from Report portal on Azure devops directly
The only way to setup this plugin https://reportportal.io/docs/plugins/AzureDevOpsBTS/ by uploading jar file but I don't know how can I get this jar file could you please help me with this?

**Comment by DzmitryHumianiuk** [comment, marked as answer]
@hager-yousri 
### Step 1 
https://reportportal.io/docs/plugins/AzureDevOpsBTS/
<img width="970" alt="image" src="https://github.com/reportportal/reportportal/assets/11332788/94568306-33ea-4092-81c1-859fbcecc1d2">

### Step 2
https://reportportal.io/docs/plugins/ManagePlugins#upload-plugin
<img width="948" alt="image" src="https://github.com/reportportal/reportportal/assets/11332788/d4226a21-c3b7-4406-9925-9ab859e6867e">

### Step 3
https://github.com/reportportal/plugin-bts-azure/packages
<img width="1301" alt="image" src="https://github.com/reportportal/reportportal/assets/11332788/5c1b1c09-3d41-445c-bd84-cdf434fdbb1e">

### Step 4
https://github.com/reportportal/plugin-bts-azure/packages/1366495
<img width="1033" alt="image" src="https://github.com/reportportal/reportportal/assets/11332788/9475ad86-30e7-41a7-8af0-d4dcf9152795">
